### ISSUE TITLE
Error: Enum case value must be compile-time evaluatable 

### ISSUE THREAD

**Issue by cawa-93** [zircote/swagger-php#1671]
I'm getting an error when try to parse project on Enum:

```php
use Webspark\Streetmedia\PartnerCabinet\Plugin;

enum FaqSettings: string implements SettingDefinition
{
    case INDEX_PAGE_ID = Plugin::NAMESPACE . '__faq_settings__index_page_id';
}
```

The `Plugin::NAMESPACE` is a `const` so AFAIK it should be compile-time evaluatable ?

**Comment by DerManoMann** [comment]
Yep, looks like a bug. Good news is that this seems to work with the 5x-dev branch :) It would be great if you could try that and let me know how that goes.

[v5] is planned to be released in the next few weeks, once PHP 8.4 is officially available.

**Comment by cawa-93** [comment]
@DerManoMann How I can try 5x-dev branch ?

**Comment by DerManoMann** [comment]
Have a look at something like this: https://mazer.dev/en/php/composer/how-to-require-specific-branch-name-composer/
